source/index.js

```diff
@@ -136,6 +136,8 @@ export { default as mergeWithKey } from './mergeWithKey.js';
 export { default as min } from './min.js';
 export { default as minBy } from './minBy.js';
 export { default as modulo } from './modulo.js';
+export { default as modify } from './modify.js';
```

Comment:
A very minor nit: these should move *before* `modulo` too keep everything alphabetic.

Reply:
Ah, definitely, it was a mistake on my part.